modules/mapper-extras/build.gradle

```diff
@@ -28,3 +28,9 @@ if (BuildParams.isSnapshotBuild() == false) {
     systemProperty 'es.index_mode_feature_flag_registered', 'true'
   }
 }
+
+tasks.named("yamlRestTestV7CompatTest").configure {
+  systemProperty 'tests.rest.blacklist', [
+    'match_only_text/10_basic/Wildcard highlighting',
```

Comment:
can this not be simplified as you just pass one string? 

meaning

```
systemProperty 'tests.rest.blacklist', 'match_only_text/10_basic/Wildcard highlighting'
```

Reply:
it makes it easier to add more blacklisted tests (it's how we blacklist in other places). Blocking this PR on a soon defunct 8.1 branch for that stylistic choice is quite concerning though.

Reply:
didn't mean too.

Reply:
@ywelsch I didn't want to offend anyone with my review comment or block anything.

Reply:
All good, I was a bit stressed out getting this merged as I'm going on PTO EOD :-)